pkg/base/k8s/cr_helper.go

```diff
@@ -342,44 +342,39 @@ func (cs *CRHelper) GetACCRs(node ...string) ([]accrd.AvailableCapacity, error)
 	return res, nil
 }
 
-// GetDriveCRByUUID reads drive CRs and returns drive CR with uuid dUUID
-func (cs *CRHelper) GetDriveCRByUUID(dUUID string) *drivecrd.Drive {
-	driveCRs, _ := cs.GetDriveCRs()
-	for _, d := range driveCRs {
-		if d.Spec.UUID == dUUID {
-			return &d
-		}
-	}
-
-	cs.log.WithFields(logrus.Fields{
-		"method":    "GetDriveCRByUUID",
-		"driveUUID": dUUID,
-	}).Infof("Drive CR isn't exist")
-	return nil
-}
-
 // GetDriveCRByVolume reads drive CRs and returns CR for drive on which volume is located
 func (cs *CRHelper) GetDriveCRByVolume(volume *volumecrd.Volume) (*drivecrd.Drive, error) {
 	ll := cs.log.WithFields(logrus.Fields{
 		"method": "GetDriveCRByVolume",
 		"volume": volume.Name,
 	})
 
-	dUUID := volume.Spec.Location
+	var (
+		dUUID = volume.Spec.Location
+		drive = &drivecrd.Drive{}
+		ctx   = context.Background()
```

Comment:
Why not passing context from upward?

Reply:
We have issue about it - https://github.com/dell/csi-baremetal/issues/703. It's need to refactor interface and many UT. I suggest to do it under separate PR to not increase changes too much